src/Symfony/Component/Form/Console/Descriptor/TextDescriptor.php

```diff
@@ -94,6 +97,29 @@ protected function describeResolvedFormType(ResolvedFormTypeInterface $resolvedF
         }
     }
 
+    protected function describeOption(OptionsResolver $optionsResolver, array $options = array())
+    {
+        $definition = $this->getOptionDefinition($optionsResolver, $options['option']);
+
+        $dump = $this->getDumpFunction();
+        $map = array(
+            'Required' => 'required',
+            'Default' => 'defaults',
+            'Allowed types' => 'allowedTypes',
+            'Allowed values' => 'allowedValues',
+            'Normalizer' => 'normalizers',
```

Comment:
isnt the normalizer an option implem. detail anyway? How is this useful, other then knowing _a_ value will be normalized.

Reply:
The normalizer information should be useful to know whether the `default` value will be modified or not, if so, the file+line should be useful for debugging purpose.

imagine a situation where the default value is `null` (or any other) and a different value is displayed in the view. Then, we need show all possible information (that can affect this value) to solve the uncertainty.

Reply:
Fair. From a debugging pov i'd really care about the actual value being used. Agree the command would then show a context where no options are passed. `--value some` might work.

Basically im in favor of enabling to pass options from CLI. To see what they resolve to.

Reply:
Not sure, `--value some` wouldn't work for all no scalar options, it would fall short :)

Reply:
Agree. Lets keep this a definition debugging tool :+1: